NEWS

```diff
@@ -4,6 +4,8 @@ smoof 1.6.0.4
 * Fixed bug in getLoggedValues when logging of x-values was set to FALSE in addLoggingWrapper
```

Comment:
General comment: given your nice contribution I suggest to add yourself to the list of contributors in DESCRIPTION 🥳